src/Raven.Client/Documents/Operations/DelayBackupOperation.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations;
+
+public class DelayBackupOperation : IMaintenanceOperation<OperationState>
```

Comment:
There is a directory Backups, it should be moved there and namespace should be adjusted?

Reply:
Done/